lib/main.coffee

```diff
@@ -26,6 +26,13 @@ openPanel = (settingsView, panelName, uri) ->
   settingsView.showPanel(panelName, options)
 
 module.exports =
+  handleUri: (parsed) ->
```

Comment:
This should be `handleURI` right?

Reply:
Ah, if we want to match conventions with Atom core, yeah (I'll make that change directly on master), but this case matches what's in `package.json`.